src/sentry/integrations/jira/integration.py

```diff
@@ -278,11 +278,15 @@ def get_group_description(self, group, event, **kwargs):
             output.extend(["", "{code}", body, "{code}"])
         return "\n".join(output)
 
-    def get_client(self):
+    def get_client(self, context=None):
```

Comment:
@ceorourke doesn't seem like `context` is being used here

Reply:
good catch, that was leftover from some other nonsense I was doing